src/index.ts

```
@@ -115,13 +115,16 @@ import {
   QuizMultipleChoice,
   QuizDissertative,
+  QuizTrueOrFalse,
+  QuizConnectDots,
   QuizQuestionList,
   QuizFooter,
   QuizListResult,
   QuizResultHeaderTitle,
   QuizResultTitle,
   QuizResultPerformance,
   QuizListResultByMateria,
+  getStatusBadge
```

ESLint

[error] Insert `,`

(prettier/prettier)